### ISSUE TITLE
broken admin page

### ISSUE THREAD

**Issue by ghost** [Project60/org.project60.sepa#352]
When I try to open the admin page for CiviSEPA (/civicrm/admin/setting/sepa) the following error message appears:

```
PHP Fatal error:  Call to undefined method CRM_Core_Config::countryLimit() in /var/www/CiviCRM_4_7_beta2/sites/default/files/civicrm/ext/org.project60.sepa/CRM/Admin/Form/Setting/SepaSettings.php on line 101
```

Tested with CiviCRM 4.7 beta 2

**Comment by nikobo** [comment]
This is part of #351

**Comment by magnolia61** [comment]
The countryLimit() fatal error is back again in 1.23 beta1
It was fixed by Scardinus (a28ed9f940ff5e62cc58c3068a2f39f9399f001d and 43430427aedf030b97c0021de50777fb8ae5b779)

**Comment by systopia** [comment]
the 1.2 branch is not tested/released for CiviCRM 4.7. I'll merge the changes into 1.3 soon

**Comment by Fabian-SYSTOPIA** [comment]
The problem seems to be fixed (using 1.2.4 alpha and 4.7.13).
